src/baxter_interface/camera.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, name):
 
         list_svc = rospy.ServiceProxy('/cameras/list', ListCameras)
         rospy.wait_for_service('/cameras/list', timeout=10)
-        if not self._id in list_svc().cameras:
+        if not self._id in ["left_hand_camera","right_hand_camera","head_camera"]:
```

Comment:
The robot will now list all 3 cameras as services, so I don't see the benefit in hard-coding the camera names here.